lib/serve.js

```diff
@@ -62,6 +65,25 @@ function generate(filename, ...args) {
         .catch(next);
 }
 
+function generateStream(filename, ...args) {
+    return (req, res, next) => gen[filename](...args)
+        .then(stream => {
+            const archive = archiver('zip');
```

Comment:
We discussed that this should be inside `static`

Reply:
Fixed